lib/foreman_remote_execution_core/fake_script_runner.rb

```diff
@@ -0,0 +1,82 @@
+module ForemanRemoteExecutionCore
+  class ScriptRunner < ForemanTasksCore::Runner::Base
```

Comment:
I wanted to make it decide which implementation to use at load time to avoid checking the env variables over and over again. But sure, I can do that